src/scala/com/github/johnynek/bazel_deps/MakeDeps.scala

```diff
@@ -15,7 +15,7 @@ object MakeDeps {
 
   private[this] val logger = LoggerFactory.getLogger("MakeDeps")
 
-  def apply(g: Command.GenerateLike): Unit = {
+  def apply(g: Command.Generate): Unit = {
```

Comment:
Wrap the whole thing in IO?

Reply:
I want to push this until I make a follow up PR that actually uses cats-effect.